Makefile.am

```diff
@@ -84,8 +84,6 @@ BUILT_SOURCES=${srcdir}/deps/boost/stage/lib/libboost_thread.a ${srcdir}/deps/so
 
 clean-local:
 	cd ${srcdir}/deps/boost && ./bootstrap.sh && ./b2 clean
-	cd ${srcdir}/deps/sofia-sip && make clean  
```

Comment:
wondering why you deleted these lines?

Reply:
in CI check, there is no target for clean.

Reply:
but I want clean-local to clean sofia-sip as well, and now it wont so I am still not following why this was necessary